lib/puppet/provider/exec/powershell.rb

```diff
@@ -100,6 +108,10 @@ def checkexe(command)
   end
 
   def validatecmd(command)
+    unless command.is_a?(String) || command.is_a?(Array)
+      raise ArgumentError, _("Command must be an Array<String>, got value of class %{klass}") % { klass: command.class }
```

Comment:
I think it would be neater to use `#{}` style interpolation here. Also I think the error message needs to be expanded slightly as you are checking for a String OR an Array.

For example:

```suggestion
      raise ArgumentError, _("Command must be a String or Array<String>, got value of class #{command.class}")
```